### PR TITLE
Create recipes page

### PR DESCRIPTION
**Pull request by arisachue**
- Create recipes page that renders a css grid of recipe card components for each recipe
- Each recipe card shows an image of the recipe and the title

Screenshot of recipes page:
Screen Shot 2022-07-06 at 3 28 25 PM

### CODE REVIEW

File: src/components/App/App.jsx
Comment (mjquigley) [inline src/components/App/App.jsx:34]: nit: Same comment as on the other PR, you could add a TODO comment here. Since this data came from a specific API, it might also be worth adding a comment with the source of this data as well both to remind yourself (and in theory other people working on this code) and to make sure you are attributing the data to it's source in case their are licensing restrictions on the data (likely not a factor here but a good habit nonetheless)

**Review by mjquigley — APPROVED** [review]

**Comment by mjquigley** [comment]
Note: you shouldn't merge this until you merge the other 2 PRs since it contains all their changes as well.

**Comment by mjquigley** [comment]
Looks good to me!
lgtm

File: src/components/App/App.jsx
Comment (arisachue) [reply src/components/App/App.jsx:34]: Got it, I added the TODO comment

**Pull request merged**
